lib/models/navigator.dart

```diff
@@ -60,6 +60,16 @@ extension type Nav(NavigatorState navigator) {
   /// {@macro models.navigator_example}
   void pop<T>([T? value]) => navigator.maybePop<T>(value);
 
+  void noTransition(Widget destination, {bool replacing = false}) {
+    final route = PageRouteBuilder(
+      pageBuilder: (context, _, __) => destination,
+      transitionDuration: Duration.zero,
+      reverseTransitionDuration: Duration.zero,
+    );
+
+    replacing ? navigator.pushReplacement(route) : navigator.push(route);
+  }
```

Comment:
I noticed a few transitions that used a `BlocProvider` with `StreamOverlayFadeIn`.

My guess is that the goal was just to have an instant transition, so we can use this method for that purpose.